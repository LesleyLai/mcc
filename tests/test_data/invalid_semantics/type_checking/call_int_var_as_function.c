int main(void)
{
  int x = 42;
  return x(2);
}
