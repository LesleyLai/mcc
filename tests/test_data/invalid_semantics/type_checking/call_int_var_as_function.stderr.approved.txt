{{filename}}:4:10: Error: called object with type 'int', which is not callable
4 |   return x(2);
  |          ^

